src/templates/fundraising/contents.swig

```diff
@@ -1,6 +1,6 @@
 <div class="row">
 	<div class="toc col-xs-12">
-		<h3 class="toc-header">Contents <small>Fundraising — Winter 2015</small></h3>
+		<h3 class="toc-header">Contents
```

Comment:
Unrelated to this PR, but wonder if you should just kill this page, as everything is probably very out of date.